src/server/src/main/java/io/cassandrareaper/resources/RequestUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ * Copyright 2022-2022 The Last Pickle Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cassandrareaper.resources;
+
+import javax.servlet.ServletRequest;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.HttpMethod;
+
+import com.google.common.annotations.VisibleForTesting;
+
+public final class RequestUtils {
+  public static final String ALLOW_ALL_OPTIONS_REQUESTS_ENV_VAR_NAME = "ALLOW_ALL_OPTIONS_REQUESTS";
```

Comment:
The more I think about this, the more I think this should just be switched on with the already used `REAPER_ENABLE_CROSS_ORIGIN` environment variable instead of having the additional flag.  Simplifies things as I think this is really needed to really use CORS with auth. 🤷

Reply:
I did indeed go ahead and make the switch here, figured that's what I'd prefer, so I'd just put it forth here to figure out.